[2023-08-30T18:56:37.308+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.323+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T18:56:37.336+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T18:56:37.338+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T18:56:37.352+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T18:56:37.357+0000] {standard_task_runner.py:57} INFO - Started process 504 to run task
[2023-08-30T18:56:37.374+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpseunfhc_']
[2023-08-30T18:56:37.392+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask install_required_packages
[2023-08-30T18:56:37.437+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.553+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 5e7fbf8b7901
[2023-08-30T18:56:37.586+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.698+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T18:56:37.711+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T18:56:37.714+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T18:56:37.734+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T18:56:39.320+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T18:56:40.031+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T18:56:40.102+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T18:56:40.118+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T18:56:40.126+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T18:56:40.128+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T18:56:40.129+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T18:56:40.131+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T18:56:43.764+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T18:56:43.871+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T18:56:44.251+0000] {subprocess.py:93} INFO - 
[2023-08-30T18:56:44.252+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T18:56:44.253+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T18:56:44.507+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T18:56:44.569+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T185637, end_date=20230830T185644
[2023-08-30T18:56:44.630+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T18:56:44.654+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:19:23.803+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:23.815+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:19:23.824+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:19:23.825+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:19:23.847+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:19:23.854+0000] {standard_task_runner.py:57} INFO - Started process 220 to run task
[2023-08-30T19:19:23.864+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmps1herbt3']
[2023-08-30T19:19:23.883+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask install_required_packages
[2023-08-30T19:19:23.950+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:24.064+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 884d86529f65
[2023-08-30T19:19:24.083+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:24.173+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:19:24.188+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:19:24.191+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:19:24.207+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:19:28.615+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:19:29.507+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:19:29.638+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:19:29.650+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:19:29.662+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:19:29.671+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:19:29.673+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:19:29.675+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:19:58.730+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:19:58.790+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:19:58.798+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:19:58.800+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:19:58.802+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:19:58.980+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:19:59.016+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T191923, end_date=20230830T191959
[2023-08-30T19:19:59.043+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:19:59.078+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:31:06.365+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.375+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:31:06.384+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:31:06.385+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:31:06.400+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:31:06.407+0000] {standard_task_runner.py:57} INFO - Started process 216 to run task
[2023-08-30T19:31:06.414+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpoohct_fa']
[2023-08-30T19:31:06.428+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask install_required_packages
[2023-08-30T19:31:06.492+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.574+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host f6bcf103ff79
[2023-08-30T19:31:06.611+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.786+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:31:06.797+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:31:06.802+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:31:06.819+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:31:09.007+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:31:11.134+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:31:11.262+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:31:11.284+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:31:11.293+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:31:11.298+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:31:11.303+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:31:11.305+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:31:32.936+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:31:33.134+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:31:33.314+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:31:33.316+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:31:33.318+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:31:33.617+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:31:33.688+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T193106, end_date=20230830T193133
[2023-08-30T19:31:33.742+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:31:33.773+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:41:18.491+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.504+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:41:18.516+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:41:18.519+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:41:18.535+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:41:18.544+0000] {standard_task_runner.py:57} INFO - Started process 275 to run task
[2023-08-30T19:41:18.547+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpcebjf5m0']
[2023-08-30T19:41:18.555+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask install_required_packages
[2023-08-30T19:41:18.592+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.716+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 81eeb4b7c8a9
[2023-08-30T19:41:18.750+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.883+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:41:18.895+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:41:18.897+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:41:18.921+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:41:25.854+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:41:33.429+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:41:33.817+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:41:33.851+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:41:33.860+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:41:33.862+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:41:33.863+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:41:33.866+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:41:48.832+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:41:48.938+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:41:49.363+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:41:49.364+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:41:49.366+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:41:49.636+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:41:49.730+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T194118, end_date=20230830T194149
[2023-08-30T19:41:49.781+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:41:49.821+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:50:05.876+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:05.899+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:50:05.923+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:50:05.924+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:50:05.963+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:50:05.980+0000] {standard_task_runner.py:57} INFO - Started process 247 to run task
[2023-08-30T19:50:06.006+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmparpm3uc2']
[2023-08-30T19:50:06.049+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask install_required_packages
[2023-08-30T19:50:10.864+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:10.998+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 206ecb1dc8df
[2023-08-30T19:50:11.033+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:11.266+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:50:11.293+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:50:11.295+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T19:50:11.415+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:50:25.609+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:50:30.715+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:50:30.866+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:50:31.248+0000] {subprocess.py:93} INFO - Collecting geoalchemy2
[2023-08-30T19:50:31.272+0000] {subprocess.py:93} INFO -   Downloading GeoAlchemy2-0.14.1-py3-none-any.whl (54 kB)
[2023-08-30T19:50:31.320+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.0/55.0 kB 1.4 MB/s eta 0:00:00
[2023-08-30T19:50:31.329+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:50:31.334+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T19:50:31.336+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T19:50:31.341+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:50:31.342+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:50:31.352+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:50:31.354+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:50:31.393+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T19:50:49.299+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy, geoalchemy2
[2023-08-30T19:50:49.372+0000] {subprocess.py:93} INFO - Successfully installed geoalchemy2-0.14.1 sodapy-2.2.0
[2023-08-30T19:50:49.691+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:50:49.692+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:50:49.693+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:50:50.001+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:50:50.118+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T195005, end_date=20230830T195050
[2023-08-30T19:50:50.181+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:50:50.219+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:54:01.067+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.078+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:54:01.093+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T19:54:01.096+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:54:01.112+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T19:54:01.119+0000] {standard_task_runner.py:57} INFO - Started process 205 to run task
[2023-08-30T19:54:01.127+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpol6rmplp']
[2023-08-30T19:54:01.144+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask install_required_packages
[2023-08-30T19:54:01.213+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.317+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 86f9e97d6c70
[2023-08-30T19:54:01.371+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.481+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T19:54:01.498+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:54:01.500+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T19:54:01.519+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:54:04.612+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:54:05.175+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:54:05.268+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:54:05.325+0000] {subprocess.py:93} INFO - Collecting geoalchemy2
[2023-08-30T19:54:05.627+0000] {subprocess.py:93} INFO -   Downloading GeoAlchemy2-0.14.1-py3-none-any.whl (54 kB)
[2023-08-30T19:54:05.673+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.0/55.0 kB 1.3 MB/s eta 0:00:00
[2023-08-30T19:54:05.678+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:54:05.688+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T19:54:05.689+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T19:54:05.701+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:54:05.702+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:54:05.704+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:54:05.705+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:54:05.789+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T19:54:23.296+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy, geoalchemy2
[2023-08-30T19:54:23.464+0000] {subprocess.py:93} INFO - Successfully installed geoalchemy2-0.14.1 sodapy-2.2.0
[2023-08-30T19:54:23.478+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:54:23.480+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:54:23.481+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:54:23.695+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:54:23.744+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T195401, end_date=20230830T195423
[2023-08-30T19:54:23.781+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:54:23.825+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:05:28.928+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:28.941+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:05:28.966+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:05:28.967+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:05:28.984+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T20:05:28.990+0000] {standard_task_runner.py:57} INFO - Started process 421 to run task
[2023-08-30T20:05:29.008+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp21att69d']
[2023-08-30T20:05:29.013+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask install_required_packages
[2023-08-30T20:05:33.814+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:37.346+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 3938d7abd9ff
[2023-08-30T20:05:37.362+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:37.526+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T20:05:37.533+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:05:37.535+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T20:05:37.570+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:05:41.594+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:05:41.878+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:05:41.879+0000] {subprocess.py:93} INFO - Requirement already satisfied: geoalchemy2 in /home/***/.local/lib/python3.8/site-packages (0.14.1)
[2023-08-30T20:05:41.881+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:05:41.885+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T20:05:41.887+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T20:05:41.891+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:05:41.893+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:05:41.894+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:05:41.895+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:05:41.936+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T20:05:45.559+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:05:45.567+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:05:45.568+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:05:45.883+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:05:45.928+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T200528, end_date=20230830T200545
[2023-08-30T20:05:45.966+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:05:45.993+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:33:29.116+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:29.141+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:33:29.151+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:33:29.152+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:33:29.183+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T20:33:29.191+0000] {standard_task_runner.py:57} INFO - Started process 239 to run task
[2023-08-30T20:33:29.198+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp8j1225x9']
[2023-08-30T20:33:29.202+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask install_required_packages
[2023-08-30T20:33:29.292+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:32.480+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 961f38e8363e
[2023-08-30T20:33:32.516+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:32.639+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T20:33:32.643+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:33:32.646+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T20:33:32.674+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:33:39.923+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:33:40.321+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:33:40.323+0000] {subprocess.py:93} INFO - Requirement already satisfied: geoalchemy2 in /home/***/.local/lib/python3.8/site-packages (0.14.1)
[2023-08-30T20:33:40.326+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:33:40.330+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T20:33:40.331+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T20:33:40.336+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:33:40.337+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:33:40.338+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:33:40.339+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:33:40.382+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T20:33:46.830+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:33:46.832+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:33:46.834+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:33:46.978+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:33:47.006+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T203329, end_date=20230830T203347
[2023-08-30T20:33:47.043+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:33:47.067+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:48:15.003+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.014+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:48:15.023+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T20:48:15.025+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:48:15.038+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T20:48:15.043+0000] {standard_task_runner.py:57} INFO - Started process 294 to run task
[2023-08-30T20:48:15.048+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmprk4178w5']
[2023-08-30T20:48:15.056+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask install_required_packages
[2023-08-30T20:48:15.072+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.145+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 8f66711a2898
[2023-08-30T20:48:15.162+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.278+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T20:48:15.281+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:48:15.283+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T20:48:15.303+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:48:16.420+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:48:16.870+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:48:16.887+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:48:16.902+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:48:16.904+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:48:16.905+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:48:16.906+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:48:21.221+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:48:21.223+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:48:21.224+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:48:21.449+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:48:21.485+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T204815, end_date=20230830T204821
[2023-08-30T20:48:21.545+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:48:21.568+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:07:24.711+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.726+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T21:07:24.739+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T21:07:24.742+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:07:24.768+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T21:07:24.773+0000] {standard_task_runner.py:57} INFO - Started process 268 to run task
[2023-08-30T21:07:24.781+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpkwalxp0z']
[2023-08-30T21:07:24.787+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask install_required_packages
[2023-08-30T21:07:24.835+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.911+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host cbc745b7550b
[2023-08-30T21:07:24.938+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:25.054+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T21:07:25.057+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:07:25.060+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T21:07:25.076+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:07:26.990+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T21:07:27.188+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T21:07:27.193+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T21:07:27.202+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T21:07:27.204+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T21:07:27.206+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T21:07:27.208+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T21:07:33.013+0000] {subprocess.py:93} INFO - 
[2023-08-30T21:07:33.017+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T21:07:33.019+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T21:07:33.405+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:07:33.458+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T210724, end_date=20230830T210733
[2023-08-30T21:07:33.491+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:07:33.519+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:16:37.371+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.381+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T21:16:37.392+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-08-30T21:16:37.393+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:16:37.417+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-28 00:00:00+00:00
[2023-08-30T21:16:37.421+0000] {standard_task_runner.py:57} INFO - Started process 310 to run task
[2023-08-30T21:16:37.431+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpkhu8w8wp']
[2023-08-30T21:16:37.438+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask install_required_packages
[2023-08-30T21:16:37.483+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.549+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-28T00:00:00+00:00 [running]> on host 5b9f14bf20e0
[2023-08-30T21:16:37.574+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.648+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-08-30T21:16:37.665+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:16:37.668+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T21:16:37.684+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:16:49.625+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T21:16:55.840+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T21:16:55.846+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T21:16:55.865+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T21:16:55.866+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T21:16:55.868+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T21:16:55.869+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T21:17:00.840+0000] {subprocess.py:93} INFO - 
[2023-08-30T21:17:00.843+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T21:17:00.844+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T21:17:01.035+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:17:01.064+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230828T000000, start_date=20230830T211637, end_date=20230830T211701
[2023-08-30T21:17:01.099+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:17:01.124+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
